include/m_common_data.h

```diff
@@ -54,24 +54,58 @@ typedef struct CommonData_unk_10098 {
 } CommonData_unk_10098; // size >= 0x10
 
 
+struct PlayState;
```

Comment:
Could you put this struct forward declaration at the top of the file with the rest?